example/spec/parallel/kinesisTests/KinesisReplaySpec.js

```diff
@@ -41,7 +41,7 @@ const {
 // records from a specified time period and triggers workflows associated
 // with the kinesis-type rules.
 describe('The Kinesis Replay API', () => {
-  const maxWaitForSFExistSecs = 60 * 3;
+  const maxWaitForSFExistSecs = 60 * 2;
```

Comment:
Although this PR lowers the startSF time from at most about a minute to seconds, there is still a significant delay in starting the AsyncOperation and it processing the stream, so I am hesitant to lower this time much more as we need to ensure that the AsyncOperation has processed the messages yet no execution is started. 

90 seconds might still be tolerable, but two minutes seemed a satisfactory impact on total runtime as compared to the rest of the suite.